app/helpers/tasklist_helper.rb

```diff
@@ -36,4 +36,12 @@ def subsection_link(subsection, log, current_user)
       subsection.label
     end
   end
+
+  def review_log_text(log)
+    if log.collection_period_open?
+      "You can #{govuk_link_to 'review and make changes to this log', "/lettings-logs/#{log.id}/review"} until #{(log.form.end_date + 1.day).to_formatted_s(:govuk_date)}.".html_safe
```

Comment:
```suggestion
      "You can #{govuk_link_to 'review and make changes to this log', "/lettings-logs/#{log.id}/review"} until #{(log.form.end_date + 1.day).to_formatted_s(:govuk_date)}.".html_safe
```

Can you change this to a route helper? Should it be `+1 day`?

Reply:
Hm, good question, am double checking, the ticket says 2nd June 2022, but maybe it should be updated on the form level

Reply:
I think whatever date it is, it belongs on the form.